### DISCUSSION TITLE
Scroll position jumps after archiving an item in projects beta

### DISCUSSION THREAD

**Discussion by mzur** [community/community#12535] [Projects and Issues]
If I scroll down in my beta project (table view) and archive an item, the view scrolls up afterwards and marks the first item again. I have to scroll all the way down to continue. Instead, the view should remember the scroll position and maybe mark the previous or next item (to the archived one) in the table.

Firefox 97.0.2 on Ubuntu 20.04.

**Comment by jwest-cox** [comment]
This occurs also when moving tasks between milestones - the scroll position jumps to the location where a given task (or issue) is moved to. If the project is large, and multiple tasks are moved - this becomes a huge barrier to an effective workflow as one spends time scrolling back and forth.

Chrome Version 103.0.5060.114.

**Reply by jwest-cox** [reply]
Interestingly, you can almost 'catch' this behaviour as a very inelegant workaround. Knocking the mouse wheel up or down just after click to move stops the auto-scroll behaviour (per move, that is).
